packages/core/src/components/LayoutDashboard/LayoutDashboard.tsx

```diff
@@ -1,13 +1,15 @@
-import { useGetLoggedInFingerprintQuery, useGetKeyQuery } from '@chia-network/api-react';
+import { useGetLoggedInFingerprintQuery, useGetKeyQuery, useFingerprintSettings } from '@chia-network/api-react';
 import { Exit as ExitIcon } from '@chia-network/icons';
 import { t, Trans } from '@lingui/macro';
 import { ExitToApp as ExitToAppIcon, Edit as EditIcon } from '@mui/icons-material';
 import { Box, AppBar, Toolbar, Drawer, Container, IconButton, Typography, CircularProgress } from '@mui/material';
-import React, { type ReactNode, useState, Suspense } from 'react';
+import { useTheme } from '@mui/material/styles';
+import React, { type ReactNode, useState, Suspense, useCallback } from 'react';
 import { useNavigate, Outlet } from 'react-router-dom';
 import styled from 'styled-components';
 
 import SelectKeyRenameForm from '../../screens/SelectKey/SelectKeyRenameForm';
+import EmojiAndColorPicker from '../../screens/selectKey/EmojiAndColorPicker';
```

Comment:
Typo causing a build failure

```suggestion
import EmojiAndColorPicker from '../../screens/SelectKey/EmojiAndColorPicker';
```